docs/en/observability/ci-cd-observability.asciidoc

```diff
@@ -17,7 +17,7 @@ OpenTelemetry.
 Using the APM Server, connect all your OpenTelemetry native CI/CD tools directly to Elastic Observability.
 
 .Architecture of CI/CD Observability with Elastic Observability
-image::images/simple-arch-observability.png[Simple architecture of CI/CD Observability]
+image::images/simple-arch-observability.png[Simple architecture of CI/CD Observability with Elastic Observability]
```

Comment:
```suggestion
image::images/simple-arch-observability.png[Simple architecture of CI/CD with Elastic Observability]
```
Remove double "Observability"?

Reply:
I just updated `alt` text to match the existing visible caption above (`.Architecture of CI/CD Observability with Elastic Observability`). I can remove from both the caption and the `alt` text.